xbmc/epg/GUIEPGGridContainer.cpp

```diff
@@ -1644,7 +1644,7 @@ GridItemsPtr *CGUIEPGGridContainer::GetPrevItem(const int &channel)
   return &m_gridIndex[channelIndex][i + m_blockOffset];
 }
 
-GridItemsPtr *CGUIEPGGridContainer::GetItem(const int &channel)
+GridItemsPtr *CGUIEPGGridContainer::GetItem(const int channel)
```

Comment:
> The "const" here does not need to match the prototype - it's not part of the API. It is a local declaration for the code inside the method, asking the compiler to forbid all modifications. It has no effect outside the method body.

 thanks for explaining. did not know this.

> Do you want me to remove it anyway?

 yes, please do - for the sake of consistency. I guess there are more people like me out there, stumbling over this kind of "mismatch".